webpack.config.babel.js

```diff
@@ -67,7 +72,7 @@ const plugins = [
     filename: fontCssFileName,
   }),
   new CleanUpStatsPlugin(),
-]
+].filter(Boolean)
```

Comment:
it's not very clear... can you push the development plugins in a "if" block instead?